api/HTMLLinkElement.json

```diff
@@ -143,6 +190,335 @@
           }
         }
       },
+      "disabled": {
```

Comment:
Content attribute needs updating too, and there are notes there that maybe should be replicated here.